h/services/search_index/_queue.py

```diff
@@ -93,10 +99,16 @@ def sync(self, limit):
             return
 
         annotation_ids = {
-            URLSafeUUID.hex_to_url_safe(job.kwargs["annotation_id"]) for job in jobs
+            URLSafeUUID.hex_to_url_safe(job.kwargs["annotation_id"])
+            for job in jobs
+            if not job.kwargs.get("force", False)
         }
-        annotations_from_db = self._get_annotations_from_db(annotation_ids)
-        annotations_from_es = self._get_annotations_from_es(annotation_ids)
+        if annotation_ids:
+            annotations_from_db = self._get_annotations_from_db(annotation_ids)
+            annotations_from_es = self._get_annotations_from_es(annotation_ids)
+        else:
+            annotations_from_db = {}
+            annotations_from_es = {}
```

Comment:
There's no point in reading the annotation from Postgres or Elastic if `force` is `True`, we aren't going to use it. If all of the jobs have `force=True` then there's no point in making a Postgres or Elastic query at all.